### ISSUE TITLE
Build failure / Vaadin 14

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/WCStoriesAddon#6]
I thought I had fixed #3 (I was able to run locally by upgrading to Vaadin 14.8.20) but:

- `mvn jetty:run` fails same error occurs when upgrading to Vaadin 14.9.0, 14.9.3
- The same error occurs when compiling the demo site in production mode with Vaadin 14.8.20
- The demos for 22.0.25, 23.2.10 and 24.0.0.alpha6 are not affected.

```
ERROR in J:\fc\addons\fc-addons-demos-14\node_modules\@types\express-serve-static-core\index.d.ts
[tsl] ERROR in J:\fc\addons\fc-addons-demos-14\node_modules\@types\express-serve-static-core\index.d.ts(99,68)
      TS1110: Type expected.
```

**Comment by paodb** [comment]
I followed different approaches (remove tsconfig file so it will be recreated, different vaadin versions, different updates to tsconfig file) trying to fix this but I couldn't find a way to make it work.

**Comment by javier-godoy** [comment]
`@types/express-serve-static-core` is a dependency of `@gugadev/wc-stories` via `@types/webpack-dev-server`.

```
npm why @types/express-serve-static-core

@types/express-serve-static-core@4.17.32
node_modules/@types/express-serve-static-core
  @types/express-serve-static-core@"^4.17.31" from @types/express@4.17.15
  node_modules/@types/express
    @types/express@"*" from @types/webpack-dev-server@3.11.6
    node_modules/@types/webpack-dev-server
      @types/webpack-dev-server@"^3.1.2" from @gugadev/wc-stories@1.0.0
      node_modules/@gugadev/wc-stories
        @gugadev/wc-stories@"^1.0.0" from the root project
  @types/express-serve-static-core@"*" from @types/connect-history-api-fallback@1.3.5
  node_modules/@types/connect-history-api-fallback
    @types/connect-history-api-fallback@"*" from @types/webpack-dev-server@3.11.6
    node_modules/@types/webpack-dev-server
      @types/webpack-dev-server@"^3.1.2" from @gugadev/wc-stories@1.0.0
      node_modules/@gugadev/wc-stories
        @gugadev/wc-stories@"^1.0.0" from the root project
```

**Comment by flang** [comment]
I was able to fix this manually installing @types/express-serve-static-core:

npm i @types/express-serve-static-core@4.17.30

**Comment by paodb** [comment]
Closed by #8
